internal/cmd/root.go

```diff
@@ -152,6 +156,30 @@ func (c *rootCommand) execute() {
 		exitCode = int(ecerr.ExitCode())
 	}
 
+	var differentBinaryError binaryIsNotSatisfyingDependenciesError
```

Comment:
nitpick: maybe `unsatisfiedDependenciesError` or similar would be a better name.